src/traces/scatter/attributes.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
             'See `y0` for more info.'
         ].join(' ')
     },
+    key: {
```

Comment:
So it would default to integer keys, but anything keyable is permissible? (e.g. `key=['Argentina', 'Afghanistan', ...]`)